tests/test_zero_data_parallel/test_shard_model_v2.py

```diff
@@ -30,12 +30,7 @@ def run_fwd_bwd(model, x, enable_autocast=False):
 
 
 def run_dist(rank, world_size, port):
-    colossalai.launch(config=CONFIG,
-                      rank=rank,
-                      world_size=world_size,
-                      host='localhost',
-                      port=port,
-                      backend='nccl')
+    colossalai.launch(config=CONFIG, rank=rank, world_size=world_size, host='localhost', port=port, backend='nccl')
 
     model = Net(checkpoint=True).cuda()
```

Comment:
test model with sharded parameters for different layers.

Reply:
Can you give me an example? I don't exactly know what you mean.

Reply:
OK, it's fine. Can you add another unitest which init model with ZeroInitContext

Reply:
Also, I just merged an MR polishing unittests. Do test with the following code.
```
from tests.components_to_test.registry import non_distributed_component_funcs

for get_components_func in non_distributed_component_funcs:
    model, train_dataloader, test_dataloader, optimizer, criterion = get_components_func()
    do_your_test()
```

Reply:
OK,  I will refactor the unit test with this code snippet.